### PR TITLE
Add consistent timeout for Vectara API calls

### PR DESCRIPTION
**Pull request by 3eif**
Adds a timeout of 60 seconds for all calls to the Vectara API

Similar to https://github.com/langchain-ai/langchain/pull/8892

### CODE REVIEW

File: langchain/src/vectorstores/vectara.ts [hunk 1]
```diff
@@ -58,6 +58,8 @@ export class VectaraStore extends VectorStore {
 
   private verbose: boolean;
 
+  private vectaraApiTimeout = 60;
```
Comment (jacoblee93) [inline langchain/src/vectorstores/vectara.ts:61]: Nit: `vectaraApiTimeoutSeconds`?

File: langchain/src/vectorstores/vectara.ts [hunk 2]
```diff
@@ -202,12 +212,12 @@ export class VectaraStore extends VectorStore {
         metadata: Record<string, unknown>;
         score: number;
       }) => [
-        new Document({
-          pageContent: response.text,
-          metadata: response.metadata,
-        }),
-        response.score,
-      ]
+          new Document({
```
Comment (jacoblee93) [inline langchain/src/vectorstores/vectara.ts:215]: I think this will fail formatting - can you run `yarn format`?

**Comment by jacoblee93** [comment]
Thank you! Two comments

**Comment by 3eif** [comment]
Thanks for the comments. I've replaced the variable name and ran the formatter.

**Comment by jacoblee93** [comment]
Thank you!

**Pull request merged**